R/lintr-deprecated.R

```diff
@@ -5,23 +5,12 @@
 #'
 #' These functions have been deprecated from lintr.
 #'
-#' - `open_curly_linter()` and `closed_curly_linter()` check that open and closed curly braces
-#'   are on their own line unless they follow an else, a comma, or a closing bracket.
-#'   Deprecated in favor of [brace_linter()].
+#' - `open_curly_linter()` (use [brace_linter()])
+#' - `closed_curly_linter()` (use `brace_linter`)
```

Comment:
consistency in linking to `[brace_linter()]`?

Reply:
I think it's normal practice to only link a reference doc the first time it's used. Helps avoid link overload. That's how e.g. Wikipedia does it: https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Linking#MOS:REPEATLINK

Reply:
Then it's still missing `()`.